backend/.env.production.example

```
@@ -0,0 +1,28 @@
+# Создать пользователя: psql -U postgres -d tasktime -f scripts/mcp-db-user.sql
+#
+# Режим запуска:
+#   HTTP transport (Docker на сервере):
+#     MCP_SERVICE_TOKEN=<token> MCP_AGENT_PASSWORD=<pwd> docker compose --profile mcp up -d mcp-flow-universe
+#
+#   Stdio (локальная разработка, SSH-туннель на БД):
+#     ssh -L 5435:localhost:5432 root@72.56.7.218 -N &
+#     ssh -L 3001:localhost:3000 root@72.56.7.218 -N &
+#     MCP_ENV=production npm run mcp
+DATABASE_URL="postgresql://flowuniverse_mcp:CHANGE_ME@localhost:5435/tasktime?schema=public"
+REDIS_URL="redis://localhost:6381"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)